appserver/ejb/ejb-connector/src/main/java/org/glassfish/ejb/config/EjbTimerService.java

```diff
@@ -68,6 +68,8 @@
 
 @Configured
 public interface EjbTimerService extends ConfigBeanProxy, PropertyBag {
+    String TYPE_NONE = "None";
+    String TYPE_DATABASE = "Database";
```

Comment:
Perhaps TYPE_DATAGRID constant can be added to list all timer types.

Reply:
The reason I included these two was, that these are special -- one is default, the other has special behaviour. There's nothing else in the code that would actually match against value DataGrid in the code, and changing the value would not affect the behavior of the server. Therefore I believe it would be misleading to bake it into API.